src/main/docs/guide/dbc/sqlMapping/sqlJsonView.adoc

```diff
@@ -1,17 +1,74 @@
 Since Micronaut Data 4.0 and Oracle23c database, an entity can be mapped to an JSON VIEW as follows:
 
-[source,java]
-----
-@JsonView("CONTACT_VIEW")
-public class ContactView
-----
+snippet::example.ContactView[project-base="doc-examples/jdbc-example", source="main", tags="record-example"]
 
 where "CONTACT_VIEW" is actual name of duality json view object in the database. It is currently supported only by the Oracle database, since version 23c.
 More about Oracle JSON VIEW can be read here https://docs.oracle.com/en/database/oracle/oracle-database/23/jsnvu/overview-json-relational-duality-views.html.
 
 Essentially, json view will be treated like mapped entity and will return JSON structure from the database and be mapped to java entity. All CRUD operations can be
 performed against json view mapped entities.
 
+Also, entity can be mapped to a JSON SUBVIEW as follows:
```

Comment:
here you refer to `SUBVIEW` and later `@JsonSubView`. Which is it?

Also when mentioning `@JsonView` and `@JsonSubview` you should use the `ann::` macro to link to the actual annotation in the javadoc. For example `ann:foo.bar.JsonView[]` (replace real package name)